test/es6/proxy-issue884.js

```diff
@@ -0,0 +1,24 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var proxyHandler = {
+    has() { WScript.Echo("has"); return true; }
```

Comment:
consider printing out the queried property name here too in case that makes debugging a future bug easier